app/src/main/java/org/thunderdog/challegram/ui/ProfileController.java

```diff
@@ -765,6 +765,34 @@ private String getProfileUsername () {
     return tdlib.chatUsername(chat.id);
   }
 
+  private long getPeerId () {
+    return user != null ? user.id : secretChat != null ? secretChat.id : chat != null ? chat.id : 0;
```

Comment:
`chat.id` isn't the same as `basicGroup.id` and `supergroup.id`, and shouldn't be returned by this method / viewable as peer ID.

Reply:
Can you clarify?

Reply:
Yes.